test/Succeed/ImportWarnings.warn

```diff
@@ -4,19 +4,19 @@ when checking the definition of f
 Issue708quote.agda:9.1-29: warning: -W[no]UnreachableClauses
 Unreachable clause
 when checking the definition of example
-RewritingEmptyPragma.agda:3.1-16: warning: -W[no]EmptyRewritePragma
-Empty REWRITE pragma
+RewritingEmptyPragma.agda:3.1-16: warning: -W[no]UselessPragma
```

Comment:
These warnings used to be about a rewrite pragma being empty, but now they're about the --rewriting option being off. Is this change intentional?

Reply:
Well, there are two reasons why the pragma is useless: no rewrite rules and --rewriting off.
I don't have a strong preference for which to report in the warning.
If you think we should keep the old warning, I can revert.
Maybe that would make sense, since being empty is a stronger reason for being useless...

Reply:
I'm not so much worried about which warning gets priority, though one other option would be to report both. The main point of my comment was that we should make sure that we have tests for both cases (missing flag and empty pragma), as well as their combination.

Reply:
Yes we have both!

Reply:
It's good for me yes!